haveclip-desktop/haveclip-desktop.pro

```diff
@@ -61,18 +61,26 @@ RESOURCES += HaveClip.qrc
 
 win32:RC_FILE = src/HaveClip.rc
 
-win32:CONFIG(release, debug|release): LIBS += -L$$OUT_PWD/../haveclip-core/release/ -lhaveclipcore
-else:win32:CONFIG(debug, debug|release): LIBS += -L$$OUT_PWD/../haveclip-core/debug/ -lhaveclipcore
-else:unix: LIBS += -L$$OUT_PWD/../haveclip-core/ -lhaveclipcore
+win32 {
+    CONFIG(release, debug|release): _DEBUG_RELEASE_DIR = release/
+    else:                           _DEBUG_RELEASE_DIR = debug/
+} # else:unix:                      _DEBUG_RELEASE_DIR =
```

Comment:
_Future improvements:_ set one [`DESTDIR`](https://doc.qt.io/qt-5.12/qmake-variable-reference.html#destdir) for both debug and release libs and add "_debug" suffix for the debug lib.

**Note:** It is possible to set dir in the parent ".pro": [`MY_LIBRARY_DEST_PATH` example](https://www.qtcentre.org/threads/18000-qmake-dependency-of-application-on-common-shared-library?p=89665#post89665).